### ISSUE TITLE
CSS: Headlines h4-h6

### ISSUE THREAD

**Issue by KariRudjord** [nlbdev/pipeline#192]
Found faults in h1-h6 in testbook. Check if it is only the testbook or if irregularities also happens in ordinary books.

Check also that h1s in frontmatter is handled correct (colophone should be placed in back of book, quotes and dedications in front of book after TOC, the rest in frontmatter should be removed).

**Comment by KariRudjord** [comment]
Textbook was wrong translated from DTBook to XHTML.

**Comment by KariRudjord** [comment]
Found following errors in headlines:

h4 should start on celle 3 (not celle 4 as now)
h5 should start on celle 3 (not celle 4 as now). Should be followed by colon (:)
h6 should start on celle 3 (not celle 4 as now). Should be followed by a line of (14) (like a border)

I have sent the headline practice to Liv Kristin (OUP) for comments. They are a bit different from the practice in NorBraille, and the OUP rules for this is a bit unclear.

**Comment by josteinaj** [comment]
Here's how headlines look now according to the automatic tests:

## the h1 headline
```xml
<row/>
<row/>
<row>⠀⠀⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉</row>
<row>⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠁</row>
<row>⠀⠀⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉⠉</row>
<row/>
```

## the h2 headline
```xml
<row/>
<row/>
<row>⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠃</row>
<row/>
```

## the h3 headline
```xml
<row/>
<row>⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠉</row>
```

## the h4 headline
```xml
<row/>
<row>⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠙</row>
```

## the h5 headline
```xml
<row/>
<row>⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠑</row>
```

## the h6 headline
```xml
<row/>
<row>⠀⠀⠀⠠⠉⠓⠁⠏⠞⠑⠗⠀⠼⠋</row>
```

**Comment by KariRudjord** [comment]
@josteinaj 
Could you fix these three headlines? h4-h6 like this:

- h4 should start on celle 3 (not celle 4 as now)
- h5 should start on celle 3 (not celle 4 as now). Should be followed by colon (:)
- h6 should start on celle 3 (not celle 4 as now). Should be followed by a line of (14) (like a border)

**Comment by josteinaj** [comment]
@KariRudjord yes I have them fixed in the code, but I'm not able to compile the code anymore. @bertfrees is looking into the compile issues.

The rules seem a bit strange to me though. Why should only h5 have colons inserted? I assume that colons should not be inserted if the headline already ends in a colon? What if a h4 ends with a colon in the source file: should that be removed automatically?

Also, isn't it a bit strange to have a border for h6 but not h5/h4?

**Comment by KariRudjord** [comment]
Actually, I've only reproduced the headline practise from NorBraille with adjustments on level 1-3 (centered, now, not left aligned with 5 indents as before). I have shown these rules to Liv Kristin for an approval, but she want to feel it in braille print. So there might be changes after she has tested it.

**Comment by KariRudjord** [comment]
In 2020 came new norwegian rules for layout. Update layout for headlines 1-6

**Comment by josteinaj** [comment]
@KariRudjord if you can send me the document then I'll add it to the collection of specifications here:

- [Norwegian Braille Specifications and Guidelines](https://github.com/liblouis/braille-specs/blob/master/norwegian/README.md)
